starter-code/index.js

```diff
@@ -1,11 +1,52 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
+  constructor() {
+    this.items = [],
+    this.length = 0
+  }
+  add(item) {
+    this.items.push(item);
+    this.length++;
+    this.items.sort((a, b) => a - b);
+  }
+  get(pos) {
+    let item = this.items[pos-1];
+    return item;
+  }
+  max() {
+    if(this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.get(this.length);
+    }
+  }
+  min() {
+    if(this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.get(1);
+    }
+  }
+  average() {
+    if(this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.items.reduce((a,b) => a + b) / this.length;
```

Comment:
sandra aquí podrías haber reutilizado la función sum()
`return this.sum() / this.length`